app/controllers/system_admin/pending_trns/base_controller.rb

```diff
@@ -17,6 +17,8 @@ def load_trainee
       end
 
       def trn_request
+        return @trn_request if defined?(@trn_request)
+
         @trn_request = trainee.dqt_trn_request
```

Comment:
does this do what we want?

```
@trn_request ||= trainee.dqt_trn_request
```

Reply:
it would normally, but `trainee.dqt_trn_request` can return `nil` so I'd rather check for the variable having been created rather than presence (so it doesn't call `trainee.dqt_trn_request` more than once)